### PR TITLE
Update example

### PR DESCRIPTION
**Pull request by karpov-kir**
Update the example to reflect the latest changes and to fix some definitions. Ad integration example will be added later.

Related to:
- https://bitmovin.slack.com/archives/C073ZNS8916/p1716455648521719?thread_ts=1716357603.237369&cid=C073ZNS8916
- https://bitmovin.atlassian.net/browse/TA-2598

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -49,56 +54,71 @@
...
+            'c3.cm.seriesName': 'Test Series Name',
+            'c3.cm.seasonNumber': 'Test Season Number',
+            'c3.cm.showTitle': 'Test Show Title',
+            'c3.cm.episodeNumber': 'Test Episode Number',
+            'c3.cm.genre': 'Test Primary Genre',
+            'c3.cm.genreList': 'Test Genre List Item 1, Test Genre List Item 2',
+            'c3.cm.utmTrackingUrl': 'https://test-utm-tracking-url',
+            // Cannot be determined reliably, but if there is a need this approximation can be used.
+            // dcType: (navigator as Navigator & { connection?: { type: string } }).connection?.type
+            //   || (navigator as Navigator & { connection?: { effectiveType: string } }).connection?.effectiveType
```
Comment (dweinber) [inline example/index.html]: We should make sure this is valid Javascript (not Typescript) in here

File: example/index.html [hunk 2]
```diff
@@ -138,7 +158,7 @@
 
       function getPlayerConfig() {
         return {
-          key: 'YOUR-PLAYER-KEY',
+          key: '<playerKey>',
```
Comment (dweinber) [inline example/index.html]: We should probably use `<PLAYER_LICENSE_KEY>` like we do in https://developer.bitmovin.com/playback/docs/getting-started-web and other places for the sake of consistency

```suggestion
          key: '<PLAYER_LICENSE_KEY>',
```

**Review by dweinber — CHANGES_REQUESTED** [review]

File: example/index.html (hunk 1 above)
Comment (karpov-kir) [reply example/index.html]: Right https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/109/commits/4a138baac41b8ddcbf4c894732f434a4c945652d. 🙂

File: example/index.html (hunk 2 above)
Comment (karpov-kir) [reply example/index.html]: Done in https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/109/commits/9117ee4137992e61b45a9dd482ab531b830bd928

**Review by dweinber — APPROVED** [review]

**Pull request merged**